### ISSUE TITLE
Feature Request: Manual setting of R, G, B Values

### ISSUE THREAD

**Issue by mechmerlin** [olivia/via-config#24]
## Problem: 

Currently, RGB Colors are set using a point and click approach. This leads to several issues

1. If a user does not have a sensitive mouse, fine tuning to the specific RGB color they want can be tedious. 
2. If you have multiple boards and want the same color on each of them, the best you can do is guestimate where you clicked. 

## Proposal

Provide manual entry fields for R, G, and B values that can be set by the user, and are also updated as a user clicks on the color chart.

**Comment by Wilba6582** [comment]
Since the color is set by Hue and Saturation (as there is a global brightness which gets applied), I don't know how setting by RGB value will work, since this would also be setting the brightness. Perhaps implement Hue and Saturation edit boxes instead, with range 0-255 as this is what gets sent to the firmware and stored.

**Comment by iaman** [comment]
@Wilba6582 That sounds like a fundamentally solid approach; the core of the issue seems to be that the current picker lacks a desired level of precision for the fussiest among us.

Of course, since I don't have a keyboard that is currently compatible with the tool (but really appreciate the work y'all are doing on it), I don't have a means of making and testing a PR to handle this myself.

**Comment by drashna** [comment]
@Wilba6582 I'm assuming you mean 360 for hue and 255 for sat, since hue is degrees. :)

**Comment by Wilba6582** [comment]
@drashna no, I mean 255 for both, hue is 8 bit internally so that the effects can utilize wrapping (i.e. automatic mod 255 when adding hue values). The loss of precision between 360 and 255 is negligible.

**Comment by drashna** [comment]
>hue is 8 bit internally

Then that is a big problem.  For hue, you NEED to be able to go up to 360, or you're cutting out 30% of the color wheel. 

For instance, Hue:255 is a violet blue (more on the blue side), while Hue:360 is red. 

Additionally, QMK uses a 16 bit value for hue, because it does need that additional range.

**Comment by Wilba6582** [comment]
I meant, my RGB code uses an 8-bit value for hue for the full hue range. It's not cutting out anything, it's just scaled, the HSV to RGB function just uses hue in 0..255 range instead of 0..359

**Comment by Wilba6582** [comment]
@drashna see here: 

https://github.com/qmk/qmk_firmware/blob/master/quantum/color.c
